client/TracySysTrace.cpp

```diff
@@ -265,7 +265,7 @@ void WINAPI EventRecordCallbackVsync( PEVENT_RECORD record )
 
 static void SetupVsync()
 {
-#if _WIN32_WINNT >= _WIN32_WINNT_WINBLUE
+#if _WIN32_WINNT >= _WIN32_WINNT_WINBLUE && !__MINGW32__
```

Comment:
Please rewrite `!__MINGW32__` into a more verbose and explicit form. I don't know if this `!` is a negation requiring `__MINGW32__` to exist and how it will work when the macro is not defined (which is the main use case to consider here).

Reply:
Updated to `!defined(__MINGW32__)` which should be more explicit; we care about whether or not the __MINGW32__ define exists, not what its value is.